Formula/unisonlang.rb

```diff
@@ -0,0 +1,76 @@
+require "language/node"
+
+class Unisonlang < Formula
+  desc "Friendly programming language from the future"
+  homepage "https://unison-lang.org/"
+  url "https://github.com/unisonweb/unison.git",
+      tag:      "release/M3",
+      revision: "97cce8d23147df30b0f80ae745968db09f4e1a44"
```

Comment:
Do we need a git clone? Can we use a tarball?

Reply:
https://github.com/unisonweb/unison/archive/refs/tags/release/M3.tar.gz should work

Reply:
It does need to be a git clone, actually. The version info the gets embedded in the executable is derived from git state. If you build it from just a tarball, `ucm version` just prints out garbage instead of giving you an actual version number.

Reply:
🤦🏻‍♂️

Reply:
The formula knows the version, though. Are there command line arguments we can pass to populate the version correctly in lieu of a git repo?